src/schema/index.ts

```diff
@@ -76,6 +76,7 @@ import { text } from './nodes/text'
 import { tocElement } from './nodes/toc_element'
 import { tocSection } from './nodes/toc_section'
 import { Marks, Nodes } from './types'
+import {comment_list} from "./nodes/comment_list";
```

Comment:
One more thing, seems like we use camelCase. Also please check linting. There are no spaces around the comment_list